rollup.config.js

```diff
@@ -38,6 +39,7 @@ export default [
       }),
       json(),
       builtins(),
+      postcss({extract: 'dist/webexWidgets.css'}),
```

Comment:
How is `postcss` different from `scss` that we use in the component repo?

Reply:
didn't think it would be necessary to scss plugin since this is pure `CSS` for now.

Reply:
Why is [sass-loader](https://github.com/webex/widgets/blob/master/package.json#L77) installed as a dev dependency then?

Reply:
yeah, that could be removed. I think I added it as part of repo initiation.